lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -609,32 +579,11 @@ def get_text(self, course):
 
 
 @attr('shard_1')
-class TestNonRegisteredUser(TestCase):
-    """
-    Tests nonregistered (auto-created) users
-    """
-    def setUp(self):
-        self.request_factory = RequestFactory()
-        self.user = NonRegisteredUserFactory()
-        self.course_id = "course/id/doesnt_matter"
-
-    def test_nonregistered_user_factory(self):
-        self.assertTrue(self.user.profile.nonregistered)
-
-    def test_nonregistered_progress_404(self):
-        with self.assertRaises(Http404):
-            req = self.request_factory.get(reverse('progress', args=[self.course_id]))
-            req.user = self.user
-            views.progress(req, self.course_id)
-
-
-@attr('shard_1')
 class StartDateTests(ModuleStoreTestCase):
     """
     Test that start dates are properly localized and displayed on the student
     dashboard.
     """
-    MODULESTORE = TEST_DATA_MIXED_TOY_MODULESTORE
```

Comment:
JimC had added this for some of the split mongo stuff a while back;
we don't need it now.